filters/ratelimit/ratelimit.go

```diff
@@ -0,0 +1,130 @@
+/*
+Package ratelimit provides filters to control the rate limitter settings on the route level.
+
+For detailed documentation of the ratelimit, see https://godoc.org/github.com/zalando/skipper/ratelimit.
+*/
+package ratelimit
+
+import (
+	"time"
+
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/ratelimit"
+)
+
+// RouteSettingsKey is used as key in the context state bag
+const RouteSettingsKey = "#ratelimitsettings"
+
+type spec struct {
+	typ        ratelimit.Type
+	filterName string
+}
+
+type filter struct {
+	settings ratelimit.Settings
+}
+
+// NewLocalRatelimit creates a local measured rate limiting, that is
+// only aware of itself. If you have 5 instances with 20 req/s, then
+// it would allow 100 req/s to the backend.
+//
+// Example:
+//
+//    backendHealthcheck: Path("/healthcheck")
+//    -> localRatelimit(20, "1m")
```

Comment:
wondering if it would be enough to just call it "ratelimit". Are we preserving the symbol "ratelimit" for another variant of the rate limiters?

Reply:
Good question. 
I thought about having "local" prefix, because it's instance local. If you think it makes sense to drop that prefix, I am ok.

Reply:
ah! OK, i see, let's keep the prefix then.

Reply:
@szuecs I'm still not completely sure what is meant by "instance local"? Do you mean it's local to one skipper instance in case you run multiple skippers as ingress?

In that case, then I would say it's obvious, and "local" is not needed.

Reply:
@mikkeloscar yes it's instance local and I am not sure if this is "obvious". I don't like the wording "obvious", because it always depends on your context.